launch/autonomy_node.launch

```diff
@@ -7,11 +7,9 @@
         type="autonomy_node"
         args="$(arg plan)"
         output="screen" >
-      <remap from="/ant_pan_position_controller/command" to="/_original/ant_pan_position_controller/command"/>
-      <remap from="/ant_tilt_position_controller/command" to="/_original/ant_tilt_position_controller/command"/>         
-</node>
-  <node pkg="ow_autonomy"
-        name="GuardedMoveServer"
-        type="GuardedMoveServer"
-        output="screen" />
+		<remap from="/ant_pan_position_controller/command"
+           to="/_original/ant_pan_position_controller/command"/>
+    <remap from="/ant_tilt_position_controller/command"
+           to="/_original/ant_tilt_position_controller/command"/>         
+  </node>

```

Comment:
I don't understand why the autonomy node needs to do this re-mapping? The remapping is already done in `common.launch`. I don't remember when this was checked but I don't think this is right.

Reply:
None of this my code, and I don't recall doing this edit.  

The basic problem here, and in your previous comments, is that this branch has gotten out of date with OW-534 in ow_simulator.   I can see what the issues are.  I'll get on this shortly and let you know when it's ready.